doc/cupertino-getting-started.md

```diff
@@ -12,34 +12,59 @@ Uno Cupertino is an add-on package that lets you apply [Cupertino - Human Interf
 
 ## Getting Started
 
-> [!NOTE]
-> As of [Uno Platform 4.7](https://platform.uno/blog/uno-platform-4-7-new-project-template-performance-improvements-and-more/), the solution template of an Uno app has changed. There is no longer a Shared project (.shproj), it has been replaced with a regular cross-platform library containing all user code files, referred to as the **App Code Library** project. This also implies that package references can be included in a single location without the previous need to include those in all project heads.
+### Creating a new project with Uno Cupertino
```

Comment:
Would be nice I think @kazo0  to point to the get started doc around the first step here:
https://github.com/unoplatform/uno.extensions/blob/0c9d21a7511c75c30abba36e5b2dfabf40535ddc/doc/GettingStarted.md?plain=1#L10

With a note like in some other pages of the doc with:

```
> [!NOTE]
> Make sure to setup your environment first by [following our instructions](xref:Uno.GetStarted.vs2022).
```